release-please-config.json

```diff
@@ -1,8 +1,24 @@
 {
-  "initial-version": "0.1.0",
+  "bootstrap-sha": "2bb9ae256af30ce27837c71b34da0705604186a3",
   "packages": {
-    ".": {},
-    "tools": {}
+    "gax": {
+      "changelog-path": "CHANGELOG.md",
+      "release-type": "node",
+      "bump-minor-pre-major": false,
+      "bump-patch-for-minor-pre-major": false,
+      "draft": false,
+      "prerelease": false
```

Comment:
If these are common for all the projects, I believe you can put them at the root level and they will be inherited to all the package-level configs.